package.json

```diff
@@ -17,16 +17,16 @@
   },
   "homepage": "https://github.com/evojam/tslint-rules#readme",
   "dependencies": {
-    "tslint-eslint-rules": "5.1.0",
-    "tslint-immutable": "4.5.4",
-    "tslint-react": "3.5.1"
+    "tslint-eslint-rules": "5.4.0",
+    "tslint-immutable": "4.7.0",
+    "tslint-react": "3.6.0"
   },
   "devDependencies": {
-    "tslint": "^5.7.0",
-    "typescript": "^2.0.0"
+    "tslint": "^5.11.0",
+    "typescript": "^3.0.3"
   },
   "peerDependencies": {
-    "tslint": "^5.7.0",
-    "typescript": "^2.0.0"
+    "tslint": "^5.11.0",
+    "typescript": "^3.0.3"
```

Comment:
Peer dependency is the lowest version which is compatible with our lib. @kpudlik correct me if I'm wrong

Reply:
My bad if so. I thought that it's independent from project deps

Reply:
> is the lowest version which is compatible

Well it actually depends on `~`, `^` etc :D 
`peerDependency` just means that consumer app needs to have it installed in exact version or in any matching combination if you use `^` or `~`

Reply:
So in your case installing will fail if consumer project has `2.X.X` version of TS because you are forcing consumer app to have at least `3.0.0`